master/networking/vxlan-ipip.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Connecting workloads that you do not control
+---
+
+### Big Picture
+
+Enable inter workload communication across networks that are not workload aware.
+
+### Value
+
+Calico supports encapsulation so you can send traffic between workloads, without requiring the underlying network to be aware of pod IP addresses.
+
+### Features
+
+This how-to guide uses the following features: 
+
+- IP in IP encapsulation
+- VXLAN encapsulation
+
+### Concepts
+
+#### Routing workload IP addresses
+When a network is **aware of workload IP addresses** (either through static routes, BGP route distribution, or another mechanism), it can directly route traffic between workloads. However, not all networks are able to route workload IP addresses. For example, public cloud environments where you don’t own the hardware, AWS across VPC subnet boundaries, and other scenarios where you cannot peer Calico over BGP to the underlay or easily configure static routes. This is why Calico supports encapsulation, so you can send traffic between workloads without requiring the underlying network to be aware of workload IP addresses.
+
+#### Encapsulation types
+Calico supports two types of encapsulations: VXLAN and IP in IP.  VXLAN is supported in some environments where IP in IP is not (for example, Azure). However, VXLAN has a slightly higher per-packet overhead because the header is larger.
+
+####
+Cross subnet
+Encapsulation of workload traffic is typically required only when traffic crosses a router (crossing an IP subnet boundary). For IP in IP, Calico can perform encapsulation on: all traffic, no traffic, or only on traffic that crosses a subnet boundary.
```

Comment:
Let's adjust this line a tiny bit - "typically required only when traffic crosses a router _that is unable to route workload IP addresses on its own_"

Reply:
better. fixed.